eservice/lib/libpdo_enclave/contract.edl

```diff
@@ -21,7 +21,8 @@ enclave {
     trusted {
         //
         public pdo_err_t ecall_CreateContractWorker(
-            size_t inThreadId
+            size_t inThreadId,
+            [in, string] const char *inSerializedEnclavePolicy
```

Comment:
I was not expecting the policy as input in worker interface, but rather retrieved by a worker, for instance through an ocall from `enclave_policy.cpp`. In fact the worker is only there to hide to initialization latency of the interpreter.

Reply:
This could work. I didn't want to introduce any new ocalls, so I opted for passing in the policy when we initialize the worker. But since we do this only once, if there's no issue with adding a new ocall, then I'd be happy to add that. Is probably a bit more intuitive this way.

Reply:
(not sure which one you're saying is more intuitive, but it's a minor thing for me now. @cmickeyb , any preference between passing the policy through the worker API vs. grabbing it with an ocall?)

Reply:
I meant having a separate ocall for passing the policy into the enclave would probably be more intuitive. So yes, if this is something that we are ok with adding, I'll make the changes.

Reply:
i have no preference though it seems just packing up everything into one JSON blob is pretty easy.

Reply:
The policy is already passed into the enclave as a single json blob when we initialize the worker. If what we have now is ok, I can also defer making these changes to the next PR.